.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "mocha": true,
```

Comment:
instead of one single global eslint config, i would recommend to use cascading to separate linting of impl and tests (even tests and functional tests)
https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy

Reply:
I'll have a look.
What's the benefit of using cascading confguration?

Reply:
The great benefit is that we can control the code style better. Test cases for example have additional globals (e.g. ```describe```) which should not appear in general implementation.  

If it's to much effort for now we could perform this separation in a later phase...

Reply:
ok i guess we can start with how it is now.